frontend/src/js/components/gmedits/ColormapField.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+
+var ColormapField = React.createClass({
+    propTypes: {
+        defaultValue: React.PropTypes.string,
+        handleChange: React.PropTypes.func
+    },
+    render() {
+        return (
+            <div>
+                <h5>Colormap: </h5>
+                <select name="colormap"
```

Comment:
This may be contradicting something I said previously, but you should be getting this data from the server.